.eslintrc.js

```diff
@@ -0,0 +1,93 @@
+// default configuration tuned for development
+module.exports = {
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+      ecmaVersion: 2018,
+      sourceType: "module",
+    },
+    plugins: ["@typescript-eslint", "json", "react", "react-hooks"],
+    env: {
+      browser: true,
+      es6: true,
+      jest: true
+    },
+    settings: {
+      react: {
+        pragma: "React",
+        version: "detect"
+      }
+    },
+    ignorePatterns: [
+      "build/", "node_modules/"
+    ],
+    extends: [
+      "eslint:recommended",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:eslint-comments/recommended",
+      "plugin:json/recommended",
+      "plugin:react/recommended",
+      "plugin:react-hooks/recommended"
+    ],
+    rules: {
+      "@typescript-eslint/explicit-module-boundary-types": "off",
+      "@typescript-eslint/no-confusing-non-null-assertion": "error",
+      "@typescript-eslint/no-empty-interface": "off",
+      "@typescript-eslint/no-explicit-any": "off",
+      "@typescript-eslint/no-require-imports": "error",
+      "@typescript-eslint/no-shadow": ["error", { builtinGlobals: false, hoist: "all", allow: [] }],
+      "@typescript-eslint/no-unused-vars": ["error", { args: "none", ignoreRestSiblings: true }],
```

Comment:
```suggestion
      "@typescript-eslint/no-unused-vars": ["warn", { args: "none", ignoreRestSiblings: true }],
```
We've decided on other projects that this should be a warning.

Reply:
Fixed.